static_src/main.js

```diff
@@ -56,9 +56,10 @@ function overview() {
 function org(orgGuid) {
   orgActions.toggleSpaceMenu(orgGuid);
   orgActions.fetch(orgGuid);
+  cfApi.fetchSpaces().then(() => spaceActions.fetchAllForOrg(orgGuid));
```

Comment:
Yea I agree, this is the second of times I've had to do this and I really require something more for it. It is a case where putting the CF api calls in the actions and then returning the promise makes sense, but then I've also seen advice not to do that (although I don't remember why). But we definitely need a way of do this action first and then do this action.